src/components/molecules/PlanRoom/RoomTables/TableComponentsType/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react';
+
```

Comment:
We don't place typings at the component level, do it instead on `models` directory level.

Reply:
without adding this it will give me an error